policies.go

```diff
@@ -904,10 +904,10 @@ func (d *dcAwareRR) Pick(q ExecutableQuery) NextHost {
 // a different rack, before hosts in all other datercentres
 
 type rackAwareRR struct {
+	lastUsedHostIdx uint64
```

Comment:
Could you please add a comment so that we don't move this in the future? For example:

```go
	// lastUsedHostIdx keeps the index of the last used host.
	// It is accessed atomically and needs to be aligned to 64 bits, keep it first in the struct.
	lastUsedHostIdx uint64
```

Reply:
Done.